pkg/controller/replicaset/replica_set.go

```diff
@@ -289,6 +289,7 @@ func isReplicaSetMatch(pod *api.Pod, rs *extensions.ReplicaSet) bool {
 // When a pod is created, enqueue the replica set that manages it and update it's expectations.
 func (rsc *ReplicaSetController) addPod(obj interface{}) {
 	pod := obj.(*api.Pod)
+	glog.V(4).Infof("Pod %s created: %+v.", pod.Name, pod)
```

Comment:
The rc logs pods.labels, do we really need the whole pod? is there something in the annotations we need? if not I'd prefer this also just logged labels to reduce log spam